### ISSUE TITLE
tweak of Reverse Palette 

### ISSUE THREAD

**Issue by swhobbit** [pmattes/x3270#49]
In the "can't leave well enough alone" department, I notice when using the reverse color palette some colors are washed out against the white background.  

![x3270 old Reverse pallette colors](https://user-images.githubusercontent.com/16212073/190873626-3ce26e2f-1683-4376-8c31-d6beed44dda5.png)

Submitted for your approve is a tweaked reverse palette.

![x3270 new Reverse colors](https://user-images.githubusercontent.com/16212073/190873646-801d815c-602c-444f-b8ef-c638e49ba500.png)

*** CORRECTED VERSION ***

[new-reverse-pallette-resource.txt](https://github.com/pmattes/x3270/files/9592507/new-reverse-pallette-resource.txt)

**Comment by pmattes** [comment]
I agree and will make the changes, but with some very small tweaks.
* I left RED as firebrick. DarkRed is pretty much brown.
* I kept DEEPBLUE as blue3, so there is some (any) difference between it and plain old BLUE.
* I darkened the highlight background just a smidge. YELLOW has slightly less contrast, but I found the very light background to be nearly invisible for small areas.
* I found the wrong host color for protected/highlighted at the same time you were fixing it.
